File: IronVault/IronVault.Model/Models/NarudzbaStavka.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class NarudzbaStavka
    {
        public int NarudzbaStavkaId { get; set; }

        public int NarudzbaId { get; set; }

        public int SuplementId { get; set; }

        public Suplement Suplement { get; set; }

        public int Kolicina { get; set; }
    }
}

File: IronVault/UI/ironvault_mobile/lib/models/narudzba_stavka.dart
import 'package:ironvault_mobile/models/suplement.dart';
import 'package:json_annotation/json_annotation.dart';

part 'narudzba_stavka.g.dart';

@JsonSerializable()
class NarudzbaStavka {
  int? narudzbaStavkaId;
  int? narudzbaId;
  int? suplementId;
  Suplement? suplement;
  int? kolicina;

  NarudzbaStavka({this.narudzbaStavkaId});

  factory NarudzbaStavka.fromJson(Map<String, dynamic> json) =>
      _$NarudzbaStavkaFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$NarudzbaStavkaToJson(this);
}
